build_config/webpack.dev.conf.js

```diff
@@ -1,43 +1,50 @@
-var ExtractTextPlugin = require("extract-text-webpack-plugin");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
This library seems to be popular and more-commonly used in webpack builds, and was more effective at producing the `main.css` file for the app to ingest.